### PR TITLE
Add Random Metronome as a Tier

### PR DESCRIPTION
**Pull request by Torpidwaters**
-Requested by Erica

### CODE REVIEW

**Comment by panpawn** [comment]
> Must add "Add Random Metronome as a Tier Part 2" as well which has the necessary changes to scripts.js to function properly

Again, I have told you countless times now, do not make more than one pull request for the same thing - also, please run `npm test` before submitting a pull request...

**Comment by Torpidwaters** [comment]
Once again, nobody has showed me how to properly do this, despite having asked many times.  I would love to be able to do these all in 1 pr

**Comment by panpawn** [comment]
Good, now squash

File: data/scripts.js [hunk 1]
```diff
@@ -731,6 +731,121 @@ exports.BattleScripts = {
...
+		let hasDexNumber = {};
+		let formes = [[], [], [], [], [], []];
+		// Pick six random pokemon--no repeats, even among formes
+		// Also need to either normalize for formes or select formes at random
+		// Unreleased are okay but no CAP
+		let num;
+		for (let i = 0; i < 6; i++) {
+			do {
+				num = this.random(721) + 1;
+			} while (num in hasDexNumber);
```
Comment (panpawn) [inline data/scripts.js]: im pretty sure this isn't how we do this

**Comment by Torpidwaters** [comment]
Oh damn that worked!  Aight I can do it from now on

File: data/scripts.js (hunk 1 above)
Comment (Torpidwaters) [inline data/scripts.js]: it isn't how you do what?

**Comment by Torpidwaters** [comment]
Cleaned up the code to shorten it.  Sorry for the commit spam, was way too tired and shoulda just waited till morning.

**Comment by supersoniko** [comment]
https://www.youtube.com/watch?v=7IfkL8swmFw
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
Please squash the commits to 1 (with a proper name), if you still can't figure it out; let me know on wisp!
~Nii Sama

**Pull request merged**